v22.2/aws-dms.md

```diff
@@ -22,6 +22,18 @@ Complete the following items before starting this tutorial:
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.Creating.html) in AWS.
 - Configure a [source endpoint](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html) in AWS pointing to your source database.
 - Ensure you have a secure, publicly available CockroachDB cluster running v22.1.0 or later.
+    - If your CockroachDB cluster is running v22.2.3 or later, set the following session variables using [`ALTER ROLE ... SET {session variable}`](alter-role.html#set-default-session-variable-values-for-a-role):
```

Comment:
* I think you'd want to make note here that you'd need to create a user for DMS that you'll use for your replication endpoint. This session variable is only going got be available for v22.2.4 or higher.
* It may be worth noting that without doing this, you may end up with a DMS failure for larger tables (in some ways it is optional). Maybe worth adding as a troubleshooting step for later as well.
* This will probably apply to v22.1 as well.

Reply:
> This will probably apply to v22.1 as well at the next point release for it (looking like v22.1.14+)

Will `copy_from_atomic_enabled` be backported? I wasn't able to find info about this.

Reply:
ah my bad, `copy_from_atomic_enabled` already exists in v22.2.0. turning that off improves things but doesn't fix it unless retries are enabled.

`copy_from_atomic_enabled` doesn't exist in v22.1, so only the `copy_from_retries_enabled` setting is required.

thanks for dealing with the complexity around all this!

Reply:
@otan Updated, PTAL. I also updated the minimum recommended version on the v22.2 doc to `v22.2.0`; I think this may have been incorrect previously?